tests.py

```diff
@@ -578,72 +579,43 @@ def test_vendoring():
         stderr=subprocess.STDOUT,
     ) == 0
 
-    # Check QT_PREFERRED_BINDING_JSON errors as expected
-    print("Testing invalid QT_PREFERRED_BINDING_JSON is suppressed..")
+    #
+    # Test invalid json data
+    #
     env = os.environ.copy()
-    if 'QT_VERBOSE' in env:
-        env.pop('QT_VERBOSE')
-    # Set invalid json data
     env["QT_PREFERRED_BINDING_JSON"] = '{"Qt":["PyQt5","PyQt4"],}'
-    # Allow us to check for a binding other than None
-    env["QT_PREFERRED_BINDING"] = os.pathsep.join(("PyQt5", "PyQt4"))
-    # Check that
+
     cmd = "import myproject.vendor.Qt;"
     cmd += "import Qt;"
     cmd += "assert myproject.vendor.Qt.__binding__ != 'None', 'vendor';"
     cmd += "assert Qt.__binding__ != 'None', 'Qt';"
 
-    # Make sure we were able to import both Qt modules
     popen = subprocess.Popen(
         [sys.executable, "-c", cmd],
         stdout=subprocess.PIPE,
-        stderr=subprocess.STDOUT,
+        stderr=subprocess.PIPE,
         cwd=self.tempdir,
         env=env
     )
-    out, err = popen.communicate()
-    # Check that the python subprocess exited cleanly.
-    if popen.returncode != 0:
-        print(out)
-        msg = 'A exception was raised'
-        assert popen.returncode == 0, msg
-
-    # Check that nothing was logged if not in verbose mode.
-    error_check = b"JSONDecodeError: Expecting property name"
-    assert error_check not in out, out
 
-    print("Testing invalid QT_PREFERRED_BINDING_JSON shows in verbose..")
-    # We should get debug info if using verbose mode
-    env["QT_VERBOSE"] = "True"
-    print('+++++++')
-    print(cmd)
-    print('+++++++')
-    popen = subprocess.Popen(
-        [sys.executable, "-c", cmd],
-        stdout=subprocess.PIPE,
-        stderr=subprocess.STDOUT,
-        cwd=self.tempdir,
-        env=env
-    )
     out, err = popen.communicate()
-    # Check that the python subprocess exited cleanly.
+
     if popen.returncode != 0:
         print(out)
-        msg = "A exception was raised"
+        msg = "An exception was raised"
         assert popen.returncode == 0, msg
 
+    error_check = b"Qt.py [warning]:"
+    assert err.startswith(error_check), err
+
     print('out------------------')
     print(out)
 
     print('err ------------------')
     print(err)
```

Comment:
Wow, I left a lot of debug prints from trying to debug my tests. These can probably be removed. I don't think they are useful anymore.

Reply:
It's fine. Let's blame it on corona.